pkg/apis/submariner.io/v1/types.go

```diff
@@ -228,7 +228,7 @@ type GlobalEgressIPConditionType string
 
 const (
 	GlobalEgressIPAllocated GlobalEgressIPConditionType = "Allocated"
-	GlobalEgressIPConflict  GlobalEgressIPConditionType = "Conflict"
+	GlobalEgressIPUpdated   GlobalEgressIPConditionType = "Updated"
```

Comment:
The GitHub diff UI might be tricking me, but it seems like you're loosing `GlobalEgressIPConflict` here? I also see it doesn't seem to be used elsewhere, which I guess is why nothing breaks, but that also seems weird because I imagined an unused variable would be flagged. :confused:

Reply:
It was unused and I don't think we'll have a use for it. It's not flagged b/c it's public/exported so the linter can 't know if it's used or not.

Reply:
Okay, make sense. Might be good to flag the removal in the commit message IMHO.